cves/CVE-2002-0661.yml

```diff
@@ -131,12 +148,16 @@ major_events:
 
     The event doesn't need to be directly related to this vulnerability, rather,
     we want to capture what the development team was dealing with at the time.
-  answer:
+  answer: |
+    During the time of the vulnerability's existence, the development team was 
```

Comment:
Maybe change this to "While this vulnerability was present in the system" or something like that. First sentence is currently a little clunky

Reply:
Agreed